packages/cashier/src/Components/Confirm/__tests__/payment-agent-transfer-confirm.spec.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+import PaymentAgentTransferConfirm from '../payment-agent-transfer-confirm';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+beforeAll(() => {
+    const portal_root = document.createElement('div');
+    portal_root.setAttribute('id', 'modal_root');
+    document.body.appendChild(portal_root);
+});
+
+afterEach(cleanup);
+
+describe('<PaymentAgentTransferConfirm />', () => {
+    const amount = 10;
+    const currency = 'testCurrency';
+    const description = 'testDescription';
+    const error = {
+        code: 'testCode',
+        message: 'testMessage',
+    };
+    const header = 'Please confirm the transaction details in order to complete the transfer:';
+    const requestPaymentAgentTransfer = jest.fn();
+    const setIsTryTransferSuccessful = jest.fn();
+    const transfer_to = 'test';
+
+    it('component should be rendered', () => {
```

Comment:
@artsemkavaleu-deriv From line 32 to 48 you wrote 4 different tests for the same condition `<PaymentAgentTransferConfirm transfer_to={transfer_to} />`, isn't it better to just write one test with multiple expects?

Reply:
Done, thx for notice!